src/plugin.js

```diff
@@ -171,6 +176,19 @@ const onPlayerReady = (player, options) => {
     error = videojs.mergeOptions(error, options.errors[error.code || 0]);
 
     if (error.message) {
+      /** This check is made because we want to check if we have got a custom error
+       * message and we do not want to change that
+       */
+      let errmsg = 'The media could not be loaded, either because the server or network' +
```

Comment:
Agreed. There should be a much DRYer way to customize error messages in this plugin.